scripts/ingests/ingest_bard14_spectra.py

```diff
@@ -0,0 +1,116 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+from astropy.table import Table
+from astropy.io import ascii
+import astropy.units as u
+from astropy.coordinates import Angle
+   
+
+SAVE_DB = False  # True: save the data files(json) in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+#bard14 url : https://docs.google.com/spreadsheets/d/11o5NRGA7jSbHKaTNK7SJnu_DTECjsyZ6rY3rcznYsJk/edit#gid=0
+
+SHEET_ID = '11o5NRGA7jSbHKaTNK7SJnu_DTECjsyZ6rY3rcznYsJk'
+SHEET_NAME = 'bard14'
+full = 'all'
+
+url = f'https://docs.google.com/spreadsheets/d/{SHEET_ID}/gviz/tq?tqx=out:csv&sheet={full}'
+
+bard14_table = ascii.read(
+    url,
+    format="csv",
+    data_start=1,
+    header_start=0,
+    guess=False,
+    fast_reader=False, 
+    delimiter=",",
+)
+
+#print result table
+print(bard14_table.info)
+
+
+#Loop through data and update spectra
+#def updating_spectra(db):
+ #   for row in bard14_table[0:19]:  
```

Comment:
This is the right way to do the loop.
```suggestion
 #   for row in bard14_table[0:19]:  
```